docs/how-to/delete-an-ensembler/delete-an-ensembler-inactive.md

```diff
@@ -0,0 +1,22 @@
+# Deleting an Ensembler with related inactive entity
+
+An Ensembler only could be deleted if there are no active ensembling jobs / router version that use the ensembler. This page show the deletion of ensembler **with related inactive entity** (router version and ensembling job).
+
+Deleting the ensembler will also delete any inactive entity related to the ensembler
+
+Navigate to Ensemblers page
+
+Click on the delete button 
+
+![](../../.gitbook/assets/ensembler_page.png)
+
+The Modal will show related entity that is going to be deleted following the deletion of the ensembler. the router version and the ensembling job shown on the popup will get deleted since both are inactive.
```

Comment:
Nit: `entity that is` -> `entities that are` (in plural, since there can be more than one)